source/common/common/base64.cc

```diff
@@ -234,6 +234,13 @@ std::string Base64::encode(const char* input, uint64_t length, bool add_padding)
   return ret;
 }
 
+void Base64::completePadding(std::string& encoded) {
+  if (encoded.length() % 4 != 0) {
+    std::string trailing_padding(4 - encoded.length() % 4, '=');
+    encoded.append(trailing_padding);
```

Comment:
use `absl::StrAppend`

Reply:
absl::StrAppend is more efficient for appending more than 1 string. It should make no difference to append only one.

Reply:
Done. Thanks!